extras/splat/splat-container-resource.js

```diff
@@ -0,0 +1,127 @@
+import {
+    BoundingBox,
+    ContainerResource,
+    Entity,
+    Vec3
+} from 'playcanvas';
+
+import { Splat } from './splat.js';
+import { SplatInstance } from './splat-instance.js';
+
+const globalDebugRender = false;
+
+class SplatContainerResource extends ContainerResource {
+    device;
+
+    splatData;
+
+    splat;
+
+    focalPoint = new Vec3();
+
+    // TODO: this can be set on render component -> mesh instance, remove this one?
```

Comment:
Yeah this should be removed and probably also `focalPoint` (and the functions below dealing with them).

The aabb should probably be stored on `Splat` (passed into constructor) then `SplatInstance` can set its `MeshInstance` aabb in constructor.

The editor will need to calcualte custom aabb when splats edited, but I will probably use SplatData directly for that.